test/support.ts

```diff
@@ -20,6 +20,27 @@ export async function submit(
   return receipt;
```

Comment:
Also here `assert.equal(receipt.status, 1, "transaction must be successful");` is dead code, it can't be ever triggered.

Reply:
What do you mean? The code runs as long as the tx doesn’t throw

Reply:
Yes, but if a tx doesn't throw, this assertion won't throw either. When a tx unexpectedly reverts, the test never prints `transaction must be successful`, an exception is thrown earlier.

Reply:
I see so you’re saying the status is always ‘1’ in cases where it doesn’t throw.

Reply:
Exactly